### ISSUE TITLE
Issue with tabs navigation if there are no second level markdown items

### ISSUE THREAD

**Issue by brendo** [squidfunk/mkdocs-material#197]
## Description

While experimenting with the new tabs feature, we encountered an error where the navigation cannot handle the scenario where there are no Markdown files at the second level, but there may be Markdown in the third level.

### Expected behavior

I would expect clicking on the Link in the navigation should navigate to the first item in that level.

### Actual behavior

Nothing happens when clicking on the Link.

### Steps to reproduce the bug

1. Assume a pages structure like the following:

```
pages:
  - Home: index.md
  - Animals:
    - Mammals:
      - Kangaroo: 'animals/mammals/kangaroo.md'
```

2. Activate the `tabs` feature:

```
feature:
  tabs: true
```

3. Preview the site with `mkdocs serve`.

4. Click on Animals.

5. Observe the page does not change (but note that you can access the page via the Footer!)

### Package versions

* Python 2.7.13
* mkdocs, version 0.16.1
* mkdocs-material 1.1.1

### Project configuration

```
site_name: Test case
site_description: Test case for tabs

theme: material

# Options
extra:
  feature:
    tabs: true
  palette:
    primary: indigo
    accent: indigo

# Extensions
markdown_extensions:
  - markdown.extensions.admonition
  - markdown.extensions.codehilite(guess_lang=false)
  - markdown.extensions.def_list
  - markdown.extensions.footnotes
  - markdown.extensions.meta
  - markdown.extensions.toc(permalink=true)
  - pymdownx.arithmatex
  - pymdownx.betterem(smart_enable=all)
  - pymdownx.caret
  - pymdownx.critic
  - pymdownx.emoji:
      emoji_generator: !!python/name:pymdownx.emoji.to_svg
  - pymdownx.inlinehilite
  - pymdownx.magiclink
  - pymdownx.mark
  - pymdownx.smartsymbols
  - pymdownx.superfences
  - pymdownx.tasklist(custom_checkbox=true)
  - pymdownx.tilde

# Page tree
pages:
  - Home: index.md
  - Animals:
    - Mammals:
      - Kangaroo: 'animals/mammals/kangaroo.md'
```

### System information

* OS: Mac Sierra
* Browser: Chrome

**Comment by squidfunk** [comment]
Didn't think of this case, should be easily fixable. I will look into this.

**Comment by squidfunk** [comment]
Fixed in #198 - please try/verify with the latest master, so I can release the bugfix as 1.1.2.

**Comment by brendo** [comment]
Thanks @squidfunk, I have tested locally and can confirm this works 👍

**Comment by squidfunk** [comment]
Fixed in release 1.2.0
